tests/test_nxviz.py

```diff
@@ -114,3 +137,20 @@ def test_init_node_colors():
     G = make_graph_for_grouping()  # noqa
     b = BasePlot(graph=G)
     assert len(set(b.node_colors)) == 1
+
+
+def test_init_edge_colors():
+    """
+    Does two checks:
+    1. If edge_color is not passed in as a keyword argument, check that
+       self.edge_colors is a list of 'blue', of length (number of nodes).
+    2. If edge_color is passed in as a keyword argument, check that
+       self.node_colors is a list with more than one element.
+    """
+    G = make_graph_for_edges()  # noqa
+    b = BasePlot(graph=G, edge_color="weight")
+    assert len(set(b.edge_colors)) > 1
+
+    G = make_graph_for_grouping()  # noqa
+    b = BasePlot(graph=G)
+    assert len(set(b.edge_colors)) == 1
```

Comment:
Not sure why this one is failing. Do you have any hypotheses as to why?

Reply:
Maybe two blank lines are needed. But I am not sure..

Reply:
If you run the test locally, does it pass?

Reply:
You mean if I can run "python test_plots.py" right? Sorry that is my first PR with someone outside of my group and with GitHub, so excuse me for being a bit slow and maybe misunderstandig things. Thank you for you time :)

Reply:
But yes when I run test_plot.py there are no complains

Reply:
No worries about that! I was at the place you’re at before :). By running it locally, I meant calling ‘py.test’ In the terminal, or by running the code in the test function manually. Could you try that out to see if things are working locally?

Reply:
Found the mistake. I am correcting it right now. Whats to to next?